models/face_detection_yunet/demo.py

```diff
@@ -32,15 +32,15 @@ def str2bool(v):
     print('This version of OpenCV does not support TIM-VX and NPU. Visit https://gist.github.com/fengyuentau/5a7a5ba36328f2b763aea026c43fa45f for more information.')
```

Comment:
Could you replace this link (and the links in other demos) with https://github.com/opencv/opencv/wiki/TIM-VX-Backend-For-Running-OpenCV-On-NPU?

Reply:
Pushed again with those changes.